.github/workflows/schedule_snapshot_release.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           body: |
             Automatic snapshot release of iree.
           draft: true
-          prerelease: true
+          prerelease: false
```

Comment:
@stellaraccident any reservations about dropping the "prerelease" label on releases and making all GitHub Releases snapshots "full"/"official" releases? We can still publish stable releases through a different channel (or multiple channels, e.g. PyPI for wheels, Maven Central for Java/Android, vcpkg for C++) with a directly controlled index page.

Reply:
Turns out the action still works if snapshots are marked as pre-release, so I reverted this line